docs/tutorials/export.md

```diff
@@ -0,0 +1,95 @@
+# Export models for inference
+
+In this section we will see how quadra allows you to export your trained models for inference. We will see how to export models for both Lightning based tasks and Sklearn based tasks.
+By default the standard export format is Torchscript, but you can also export models to ONNX or plain Pytorch (this is done for particular operations like gradcam).
+
+## Standard export configuration
+
+The standard configuration for exporting models is located at `configs/export/default.yaml` and it is shown below:
+
+```yaml
+types: [torchscript]
+input_shapes: # Redefine the input shape if not automatically inferred
+onnx:
+  # torch.onnx.export options
+  input_names: # If null automatically inferred
+  output_names: # If null automatically inferred
+  dynamic_axes: # If null automatically inferred
+  export_params: true
+  opset_version: 16
+  do_constant_folding: true
+  # Custom options
+  fixed_batch_size: # If not null export with fixed batch size (ignore dynamic axes)
+  simplify: true
+```
+
+`types` is a list of the export types that you want to perform. The available types are `torchscript`, `onnx` and `pytorch`. By default the models will be saved under the `deployment_model` folder of the experiment with extension `.pt`, `.onnx` and `.pth` respectively. Pytorch models will be saved alongside the yaml configuration for the model itself so that you can easily load them back in python.
+
+`input_shapes` is a parameter that will be `None` most of the time, quadra features a model wrapper that is capable of inferring the input shape of the trained model based on its forward function. It supports a large variety of custom forward functions where parameters are combinations of lists, tuples or dicts. However, if the model wrapper is not able to infer the input shape you can specify it here, the format is a list of tuples/lists/dicts where each element represents a single input shape without batch size. For example if your model has an input of shape (1, 3, 224, 224) you can specify it as:
```

Comment:
I believe it would be nicer to add also a more complex case with the usage of tuples/lists/dicts.